test/unit/logic/transaction_pool.js

```diff
@@ -929,13 +927,11 @@ describe('transactionPool', () => {
 			transactionPool.processUnconfirmedTransaction = sinonSandbox
 				.stub()
 				.callsArgWith(2, null, transaction);
-			expect(
-				transactionPool.receiveTransactions(transaction, {}, (err, res) => {
-					expect(err).to.be.null;
-					expect(res).to.deep.eql(transaction);
-					done();
-				})
-			);
+			transactionPool.receiveTransactions(transaction, {}, (err, res) => {
+				expect(err).to.be.null;
+				expect(res).to.deep.eql(transaction);
```

Comment:
`expect(res).to.deep.equal(transaction);`

Reply:
We don't have standard for this.